src/index.ts

```diff
@@ -102,27 +102,20 @@ client.on('messageCreate', async (message) => {
         hookData.run(client, message, log);
     });
     let msgIsCommand = false;
-    let prefixLen = 0;
-    const prefix = client.guildData.get(message.guild!.id, 'prefix');
+    const prefix = client.guildData.get(message.guild!.id, 'prefix')!;
+    const mlc = message.content.toLowerCase();
```

Comment:
```suggestion
    const lowercasedMessageContent = message.content.toLowerCase();
```
descriptive variable names please:tm:

Reply:
m

Reply:
Resolved: #1b476fc

Reply:
You know that commits are just like 1b476fc lol no #